CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v5.12.3
+* (Bugfix) Current time context field will automatically populate with UTC aware time ([#324](https://github.com/Unleash/unleash-client-python/pull/324)). Thanks @jacob-indigo!
```

Comment:
This one isn't currently listed in the releases section, but I guess this is part of that release

Reply:
Yessir!